src/main/kotlin/org/jetbrains/kotlinx/jupyter/repl.kt

```diff
@@ -420,10 +424,11 @@ class ReplForJupyterImpl(
             notebook.updateVariablesState(internalEvaluator)
             // printVars()
             // printUsagesInfo(jupyterId, cellVariables[jupyterId - 1])
+            val entry = notebook.variablesState.entries.lastOrNull()
```

Comment:
It's not clear why it should be last, especially in unordered map

Reply:
Just for the test) just grab any variable